lib/quick_settings_dialog.dart

```diff
@@ -17,15 +17,17 @@ Widget quickSettingsDialog(BuildContext context, Map songData) =>
             shrinkWrap: true,
             children: [
               const SizedBox(height: 10),
-              const SettingsSectionTitle('Kapcsolódó'),
+              if (songData["links"].isNotEmpty)
```

Comment:
Mivel a jsonban nem volna szabad üres listának lennie (mert felesleges), ezért ne azt nézzük, hogy üres-e a lista, hanem hogy van-e lista egyáltalán.

Reply:
A már meglévő kód működik egy előre instanciált lista feltöltésével, de módosítható null kezeléses megoldásra. Legyen úgy?

Reply:
Oh azt hittem másik részre vonatkozik a kérdés. És jogos is, mert itt ha null lenne, hibát dobna.

Reply:
Fixed.